backend/server/app.py

```diff
@@ -65,7 +57,7 @@ async def get_collections():
     try:
         vector_db_client = get_vector_db_client(config=settings.VECTOR_DB_CONFIG)
         collection_names = vector_db_client.get_collections()
-        collections = metadata_store_client.get_collections(
+        collections = settings.METADATA_STORE_CLIENT.get_collections(
```

Comment:
nit
this looks a little weird to write

Keep it for now

Reply:
removed